azurelinuxagent/common/protocol/restapi.py

```diff
@@ -125,6 +126,25 @@ def sort_key(self):
         return level
 
 
+# too-few-public-methods<R0903> Disabled: More public methods are not needed here.
+class InVMGoalStateMetaData(DataContract):  # pylint: disable=R0903
+    """
+    Object for parsing the GoalState MetaData received from CRP
+    Eg: <InVMGoalStateMetaData inSvdSeqNo="2" createdOnTicks="637405409304121230" activityId="555e551c-600e-4fb4-90ba-8ab8ec28eccc" correlationId="400de90b-522e-491f-9d89-ec944661f531" />
+    """
+    def __init__(self):
+        self.in_svd_seq_no = None
+        self.created_on_ticks = None
```

Comment:
I think so but let me confirm

Reply:
I saw this comment in the CRP code - 
>/// We need to keep the value "CreatedOnTicks" in the serialized SVD because the Windows Guest Agent
        /// Depends on it. We should be able to remove this when FastTrack V2 is world wide, as this will no
        /// longer be relevant then.

Since this is destined to be removed from the GoalState eventually, I'm going to remove the `created_on_ticks` property altogether from the code. We dont need it anyways right now

Reply:
Removed